pkg/controller/migplan/validation.go

```diff
@@ -654,3 +665,79 @@ func containsAccessMode(modeList []kapi.PersistentVolumeAccessMode, accessMode k
 	}
 	return false
 }
+
+type NamespacePods struct {
+	Namespace string         `json:"namespace"`
+	Pods      []PodWithPhase `json:"pods"`
```

Comment:
I'm not convinced the `phase` is useful given the short list of possible failure phases.  Given this, suggest `types.NamespacedName` is sufficient instead of a custom type.